kratos/python_scripts/from_json_check_result_process.py

```diff
@@ -140,7 +140,7 @@ def ExecuteAfterOutputStep(self):
     def ExecuteFinalize(self):
         pass
 
-    def __linear_interpolation(x, x_list, y_list):
+    def __linear_interpolation(self, x, x_list, y_list):
```

Comment:
same comment. i am approving... but i think this is a candidate for using a kratos table